internal/signaling/timeout_manager.go

```diff
@@ -58,11 +58,9 @@ func (i *TimeoutManager) disconnectPeerInLobby(ctx context.Context, peerID strin
 		Type: "disconnect",
 		ID:   peerID,
 	}
-	data, _ := json.Marshal(packet)
-
-	err := i.Store.LeaveLobby(ctx, gameID, lobby, peerID)
+	data, err := json.Marshal(packet)
 	if err != nil {
-		logger.Warn("failed to leave lobby", zap.Error(err))
+		logger.Error("failed to marshal disconnect packet", zap.Error(err))
```

Comment:
Diff here is slightly messy, but it removes the call to `Store.LeaveLobby`. It's messy because it also adds an error check to the result of `json.Marshal`.

Reply:
Leaving a lobby is not implemented yet https://github.com/poki/netlib/issues/9